### ISSUE TITLE
Autosave not saving new files

### ISSUE THREAD

**Issue by raymanoz** [atom/autosave#67]
If I start editing a new (non-existant) file, make some changes, and close atom, the file is not saved.

to reproduce:

```
$ atom newfile
 (add some text to newfile)
 (press cmd-q to quit atom)
$ ls newfile
 (notice newfile has not been created)
```

**Comment by tribbloid** [comment]
affect me as well, sublime text used to handle this nicely

**Comment by Hagith** [comment]
Doesn't work for me either. 
It's working with [save-session](https://github.com/mpeterson2/save-session)  package - but it's deprecated in favor of this package.

**Comment by raymanoz** [comment]
Also:

```
$ atom existingfile
  (make changes)
  (cmd-q to quit)
$ cat existingfile
 (notice existingfile has not been modified)
```

**Comment by ungb** [comment]
I was able to repro both issues.  The issue @raymanoz mentioned is a bug as well and should be addressed.

If you reopen Atom, your unsaved changes are still there.

These were my steps I followed to repro the issue:
```
mkdir testfolder && cd testfolder && atom newfile
(make some changes to the file)
(cmd+q to quit)
ls 
(new file hasn't been created)
```

**Comment by smashwilson** [comment]
Both of these issues are fixed and will ship in Atom 1.22-beta0.
